lms/templates/login.html

```diff
@@ -130,6 +130,8 @@
   </script>
 </%block>
 
+<%include file="forgot_password_modal.html" />
```

Comment:
Note that I didn't include the "not authenticated" check because I assume that if you get to login.html, it is because you aren't authenticated... Indeed, I can't access it on devstack if I am logged in (I get redirected to my dashboard).